slack/web/classes/blocks.py

```diff
@@ -310,7 +310,11 @@ def _validate_label_length(self):
 
     @JsonValidator(f"hint attribute cannot exceed {hint_max_length} characters")
     def _validate_hint_length(self):
-        return self.hint is None or len(self.hint) <= self.hint_max_length
+        return (
```

Comment:
Good catch - thanks!

Reply:
Could you add a unit test verifying the bug has been fixed?

```diff
diff --git a/tests/web/classes/test_blocks.py b/tests/web/classes/test_blocks.py
index 812cd5b..14fb330 100644
--- a/tests/web/classes/test_blocks.py
+++ b/tests/web/classes/test_blocks.py
@@ -584,7 +584,22 @@ class InputBlockTests(unittest.TestCase):
                     "text": "Label",
                     "emoji": True
                 }
-            }
+            },
+            {
+                "type": "input",
+                "element": {
+                    "type": "plain_text_input"
+                },
+                "label": {
+                    "type": "plain_text",
+                    "text": "Label",
+                    "emoji": True
+                },
+                "hint": {
+                    "type": "plain_text",
+                    "text": "some hint"
+                }
+            },
         ]
         for input in blocks:
             self.assertDictEqual(input, InputBlock(**input).to_dict())
@@ -603,6 +618,7 @@ class FileBlockTests(unittest.TestCase):
         }
         self.assertDictEqual(input, FileBlock(**input).to_dict())
```

Reply:
Sorry for not spotting that earlier. I'll make sure to add tests when touching from now on.